volunteer.html

```diff
@@ -70,8 +70,8 @@ <h3 class="text-center">Development</h3>
 <div class="panel">
   <div class="panel-container">
     <div class="panel-fill text-center row">
-      <a name="signup"></a><h2>Want to volunteer?</h2>
-      <div class="column m-8-12 m-offset-2-12 text-left">
+      <div class="column l-6-12 l-offset-3-12">
+        <a name="signup"></a><h2>Want to volunteer?</h2>
```

Comment:
I don't think we want these changes?

Reply:
It doesn't seem to make a difference to the output, but the markup being like this is definitely more correct.

Reply:
While I can't say exactly what it fixes, it does seem that this change was deliberate (if possibly unrelated): https://github.com/srobo/website/pull/87/commits/81a9325f1f7cff7a83ebe09ac9800521fbc1fae1 and https://github.com/srobo/website/pull/87/commits/5a1219a220d4e74aa963c6b5d7bd4c34643cc123

Reply:
The main change is the addition of a `text-left` class, but the text it meant to be centered anyway